pkgs/development/interpreters/php/default.nix

```diff
@@ -223,7 +223,7 @@ let
         bcmathSupport = config.php.bcmath or true;
         socketsSupport = config.php.sockets or true;
         curlSupport = config.php.curl or true;
-        curlWrappersSupport = (!php7) && (config.php.curlWrappers or true);
+        curlWrappersSupport = (php54) && (config.php.curlWrappers or true);
```

Comment:
Hmmm looking at this again, where is this `cfg` variable even used? Looks like it doesn't prevent the `configureFlags` (above) from being filtered... Any clue?